lexicon/bpe.py

```diff
@@ -41,6 +47,12 @@ def __init__(
             usually yes for RASR search and no for Flashlight search.
             The phonemes of the special lemmas will also be kept, therefore
             make sure there is no overlap with the BPE vocab.
+        :param skip_unk_lemmas: whether simply skip lemmas out of the BPE vocab
+            useful if you set vocab_blacklist
+        :param add_all_bpe_phonemes: If set to True, all BPE vocab will be added to lexicon phonemes,
+            otherwise, only phonemes appear in lexicon lemma will be added to the lexicon.
```

Comment:
```suggestion
        :param add_all_bpe_phonemes: If set to True, all BPE tokens will be added to lexicon as phonemes,
            otherwise, only tokens that appear in the base lexicon will be added to the output lexicon.
```